### PR TITLE
Add some Korean sites' change password URLs

### PR DESCRIPTION
**Pull request by jihoon-seo**
### Checklist
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [x] The top-level JSON objects are sorted alphabetically 
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

#### for change-password-URLs.json
- [x] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)

### CODE REVIEW

**Comment by rmondello** [comment]
Hi @jihoon-seo! I fixed the merge conflict on this, and removed GitHub. Do you agree to the Developer Certificate of Origin? https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md

**Comment by jihoon-seo** [comment]
@rmondello Yes, I agree!

File: quirks/change-password-URLs.json [hunk 1]
```diff
@@ -65,6 +73,7 @@
     "minecraft.net": "https://www.minecraft.net/profile",
     "mojang.com": "https://www.minecraft.net/profile",
     "myaccount.ea.com": "https://myaccount.ea.com/cp-ui/security/index",
+    "naver.com": "https://nid.naver.com/user2/help/myInfo.nhn?m=viewChangePasswd&lang=ko_KR",
```
Comment (rmondello) [inline quirks/change-password-URLs.json:76]: Okay, one last bit. Should we remove the `lang` parameter? Does the link work without it?

Comment (jihoon-seo) [reply quirks/change-password-URLs.json:76]: Yes it seems that the link works without it, so it will be nice if the `lang` parameter is removed.

**Comment by rmondello** [comment]
Thank you! :-)

**Pull request merged**